pool/command/command_matcher.ml

```diff
@@ -0,0 +1,44 @@
+let get_or_failwith = Pool_common.Utils.get_or_failwith
+
+let run =
+  Sihl.Command.make
```

Comment:
Make command to run for all tenants


And bulk insert on empty list fails (SQL ERROR)